<REFENTRY ID="baselib--Unwind-Resume-or-Rethrow"> <REFMETA>
<REFENTRYTITLE>_Unwind_Resume_or_Rethrow</REFENTRYTITLE>
<REFMISCINFO>Base Libraries</REFMISCINFO>
</REFMETA>
<REFNAMEDIV>
<REFNAME>_Unwind_Resume_or_Rethrow
</REFNAME>
<REFPURPOSE>
private C++ error handling method
</REFPURPOSE>
</REFNAMEDIV>

<REFSYNOPSISDIV>
<REFSYNOPSISDIVINFO>
<DATE>10 August 2010</DATE>
</REFSYNOPSISDIVINFO>
<FUNCSYNOPSIS>
<FUNCPROTOTYPE>
 <FUNCDEF>_Unwind_Reason_Code
  <FUNCTION>_Unwind_Resume_or_Rethrow</FUNCTION></FUNCDEF>
 <PARAMDEF>struct _Unwind_Exception *
  <PARAMETER>exception_object</PARAMETER></PARAMDEF>
</FUNCPROTOTYPE>
</FUNCSYNOPSIS>
</REFSYNOPSISDIV>

<REFSECT1>
<REFSECT1INFO>
<DATE>10 August 2010</DATE>
</REFSECT1INFO>
<TITLE>Description
</TITLE>
<PARA>
If the unwind was initiated due to a forced unwind,
<FUNCTION>_Unwind_Resume_or_Rethrow</FUNCTION> 
shall resume that operation, else it shall re-raise the exception.
</PARA>
</REFSECT1>
</REFENTRY>
